### PR TITLE
Add support to Restores

### PR DESCRIPTION
**Pull request by balestrinc**

### CODE REVIEW

File: hpOneView/oneview_client.py [hunk 1]
```diff
@@ -360,6 +362,18 @@ def fabrics(self):
         return self.__fabrics
 
     @property
+    def restores(self):
```
Comment (GustavoHennig) [inline hpOneView/oneview_client.py]: I'm not seeing the tests for this property.

File: examples/restores.py [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+restores = oneview_client.restores.get_all()
+pprint(restores)
+
+# Get by
+print("\nGet a Restorte by hostName equals to '{}'".format(restore['hostName']))
+restore_by_host_name = oneview_client.restores.get_by("hostName", restore['hostName'])
+pprint(restore_by_host_name)
+
+# Get by URI
+print("\nGet a Restorte by URI")
```
Comment (marikrg) [inline examples/restores.py]: Restore

File: examples/restores.py [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+restore = oneview_client.restores.restore(options)
+pprint(restore)
+
+# Get all restores
+print("Get all Restores")
+restores = oneview_client.restores.get_all()
+pprint(restores)
+
+# Get by
+print("\nGet a Restorte by hostName equals to '{}'".format(restore['hostName']))
```
Comment (marikrg) [inline examples/restores.py]: Restore

File: examples/restores.py [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+    }
+}
+
+# Try load config from a file (if there is a config file)
+config = try_load_from_file(config)
+
+oneview_client = OneViewClient(config)
+
+# A Uri for a backup to restore must be set to run this example
+uriOfBackupToRestore = "/rest/backups/example_backup_2017-04-25_195515"
```
Comment (marikrg) [inline examples/restores.py]: This variable is not following the python naming standard.

**Comment by balestrinc** [comment]
@marikrg Done!

**Pull request merged**